src/NSubstitute/SubstituteExtensions.cs

```diff
@@ -70,6 +70,39 @@ public static ConfiguredCall Returns<T>(this Task<T> value, Func<CallInfo, T> re
         }
 
         /// <summary>
+        /// Set a return value for this call. The value(s) to be returned will be wrapped in ValueTasks.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="value"></param>
+        /// <param name="returnThis">Value to return. Will be wrapped in a ValueTask</param>
+        /// <param name="returnThese">Optionally use these functions next</param>
```

Comment:
It should be "Optionally use these values next" here. I assume that it's a copypaste and we have it in the existing methods.

Reply:
Indeed this comment was taken from analogous `Task<T>` method for consistency and I missed that.